### DISCUSSION TITLE
Migration entfernen

### DISCUSSION THREAD

**Discussion by mega-stoffel** [wielebenwir/commonsbooking#1483] [Questions & Answers]
Hej!
Ich habe (endlich) die Migration von 0.9 auf das neue Plugin durchgeführt.
In den Einstellungen ist aber immer noch der Punkt "Migration" vorhanden, obwohl ich auch das alte Plugin deaktiviert und gelöscht habe.
Wäre es nicht gut, wenn dieser Menüpunkt "Migration" gar nicht erscheint? Ich weiß jetzt nicht, was da passiert, ich trau mich auf jeden Fall nicht, da drauf zu klicken, nachdem ich alle alten Objekte schon gelöscht habe.

**Comment by hansmorb** [comment]
Huhu, frohes neues Jahr! Da kannst du problemlos draufklicken, um die eigentliche Migration zu starten gibt es nochmal einen Knopf in dem Untermenü. Wir würden den Menüpunkt nach der Migration nicht entfernen, da bei einer unvollständigen Migration diese aus dem Menü wieder aktiviert werden kann. Und auch in Versionssprüngen zwischen CB2 Versionen kann der Migrationsbereich nochmal sinnvoll sein (zB. um auf eine neue Datenstruktur zu migrieren).

**Comment by mega-stoffel** [comment]
ebenfalls: Happy New Year! :-)
Ich war durch den roten Text, der ja eher eine "Gefahr" darstellt, etwas beeindruckt. Traue mich aber immer noch nicht, da drauf zu drücken, muss da erst mal noch eine bessere Backup/Recovery Strategie ausdenken.
Aber ist ja gut, dass grundsätzlich scheinbar nichts kaputt gehen kann, das beruhigt doch eher. Vielen Dank!
